api/src/main/java/ca/bc/gov/educ/eas/api/rest/RestUtils.java

```diff
@@ -324,4 +327,26 @@ public List<School> getAllSchoolList(UUID correlationID) {
       throw new EasAPIRuntimeException(NATS_TIMEOUT + correlationID + ex.getMessage());
     }
   }
+
+  @Retryable(retryFor = {Exception.class}, noRetryFor = {EasAPIRuntimeException.class}, backoff = @Backoff(multiplier = 2, delay = 2000))
+  public List<StudentMerge> getMergedStudentsForDateRange(UUID correlationID, String createDateStart, String createDateEnd) {
+    try {
+      final TypeReference<Event> refEventResponse = new TypeReference<>() {};
+      final TypeReference<List<StudentMerge>> refMergedStudentResponse = new TypeReference<>() {};
+      Object event = Event.builder().sagaId(correlationID).eventType(EventType.GET_MERGED_STUDENT_IDS).eventPayload(CREATE_DATE_START.concat("=").concat(createDateStart).concat("&").concat(CREATE_DATE_END).concat("=").concat(createDateEnd)).build();
+      val responseMessage = this.messagePublisher.requestMessage(TopicsEnum.PEN_SERVICES_API_TOPIC.toString(), JsonUtil.getJsonBytesFromObject(event)).completeOnTimeout(null, 120, TimeUnit.SECONDS).get();
+      if (responseMessage == null) {
```

Comment:
What will happen to the scenario of no merges happened between the period. Will this condition hold good ?

Reply:
Empty list is returned from the PEN Services API - should be fine.